src/clj/qbits/jet/client/http.clj

```diff
@@ -210,17 +214,22 @@
             dispatch-io?
             tcp-no-delay?
             strict-event-ordering?
+            http2-transport?
             ssl-context-factory]
      :or {remove-idle-destinations? true
           dispatch-io? true
           follow-redirects? true
           tcp-no-delay? true
           strict-event-ordering? false
           ssl-context-factory @default-insecure-ssl-context-factory
+          http2-transport? false
```

Comment:
Maybe `:transport :http2` default `transport :http1` rather than making this a bool.

Reply:
I changed the code to accept the `transport` instance instead. Initializing the http2 transport has additional configurations which I do not think belong in here.